app/assets/stylesheets/general.scss

```diff
@@ -2,6 +2,92 @@
 $error-red: #ff0101;
 $error-red-transparent: rgba(255, 1, 1, 0.5);
 
+/* General */
+body {
+  display: flex;
```

Comment:
Is a good practice to define the font-size in the body and html. By default browsers set it to 16px.
`font-size: 62.5%;` (I don't know why the use percentage, but this is the same as 10px).

This way if you want to use `em` as font-size's unity is easy to calculate the value. Example: 14px = 1.4em;

Also it helps to use the `rem` value. This is more useful for heights and paddings, etc.

Take into account that all your fonts sizes that use em could change.